fish_prompt.fish

```diff
@@ -393,16 +393,15 @@ end
 
 function __bobthefish_prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
   [ "$theme_display_ruby" = 'no' ]; and return
-  set -l ruby_version
-  if type rvm-prompt >/dev/null 2>&1
+  if which rvm-prompt >/dev/null 2>&1
     set ruby_version (rvm-prompt i v g)
-  else if type rbenv >/dev/null 2>&1
-    set ruby_version (rbenv version-name)
+  else if which rbenv >/dev/null 2>&1
+    set ruby_version (cat .ruby-version 2>/dev/null)
     # Don't show global ruby version...
-    [ "$ruby_version" = (rbenv global) ]; and return
+    set -q $RBENV_ROOT; and set rbenv_root ~/.rbenv; or set rbenv_root $RBENV_ROOT
```

Comment:
I think this is backwards: if `$RBENV_ROOT` is defined, use something that's not `$RBENV_ROOT`; if `$RBENV_ROOT` is _not_ defined, use `$RBENV_ROOT` :)